src/app/containers/Brand/index.jsx

```diff
@@ -1,22 +1,59 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
+import SkipLink from '@bbc/psammead-brand/skip-link';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { bool } from 'prop-types';
 import { ServiceContext } from '#contexts/ServiceContext';
 
+const renderScriptLink = (
+  script,
+  service,
+  variant,
+  url,
+  scriptLinkText,
+  offscreenText,
+) => (
+  <ScriptLink script={script} service={service} href={url} variant={variant}>
+    <span aria-hidden>{scriptLinkText}</span>
+    <VisuallyHiddenText> {offscreenText} </VisuallyHiddenText>
+  </ScriptLink>
+);
+
+const renderSkipLink = (service, script) => (
+  <SkipLink service={service} script={script} href="#content">
+    Skip to content
+  </SkipLink>
+);
+
 const BrandContainer = props => {
   const {
     product,
     serviceLocalizedName,
     brandSVG,
     service,
     theming,
+    script,
+    scriptLinkText,
+    scriptLinkOffscreenText,
+    variant = null,
```

Comment:
Will this `variant` be the one that the user is already on or the one they would switch to?

Reply:
It is the one they would switch to

Reply:
UPDATE: This variant is now the one that the user is currently on